cetmix_tower_yaml/wizards/cx_tower_yaml_export_wiz.xml

```diff
@@ -6,32 +6,86 @@
-                    <button
-                        string="Generate YAML file"
-                        type="object"
-                        name="action_generate_yaml_file"
-                        class="oe_highlight"
-                    />
-                    <button string="Close" special="cancel" />
-                </footer>
+                <sheet>
+                    <notebook>
+                        <page string="Settings">
+                            <group>
+                                <field name="explode_child_records" />
+                                <field name="remove_empty_values" />
+                                <field name="preview_code" />
+                            </group>
+                            <field
+                                name="comment"
+                                placeholder="Comment to be added to the beginning of exported YAML file"
+                            />
+                            <field
+                                name="yaml_code"
+                                widget="ace"
+                                options="{'mode': 'yaml'}"
+                                readonly="1"
+                                force_save="1"
+                                attrs="{'invisible': [('preview_code', '=', False)]}"
+                            />
+                        </page>
+                        <page string="Manifest">
+                            <group>
+                                <field
+                                    name="manifest_template_id"
+                                    placeholder="Select a pre-defined template"
+                                />
+                            </group>
+                            <group string="Information">
+                                <field
+                                    name="manifest_name"
+                                    placeholder="Leave empty to skip the manifest section"
+                                />
+                                <field
+                                    name="manifest_summary"
+                                    attrs="{'required': [('manifest_name','!=',False)]}"
+                                    placeholder="Short summary, 160 symbols max"
+                                />
+                                <field
+                                    name="manifest_author_ids"
+                                    widget="many2many_tags"
+                                    attrs="{'required': [('manifest_name','!=',False)]}"
+                                />
+                                <field
+                                    name="manifest_version"
+                                    placeholder="use the Major.Minor.Patch format, e.g. 1.2.3"
+                                />
+                                <field name="manifest_website" />
+                                <field name="manifest_description" />
```

Comment:
The content of this field can be really large. And may render the form barely readable. 
You should put it outside of group with a placeholder so it would have no label.
Also it's reasonable to use nested groups to optimise the layout, eg:

```xml
<group name="manifest" string="Manifest">
     <group ....>
          <field ....>
     </group>
     <group ....>
          <field ....>
     </group>
</group>
```